tests/conftest.py

```diff
@@ -5,10 +5,10 @@
 
 # Define all the constants used in the test
 # These values may change for each run.
-API_KEY = "5a73aa5a-1962-4df9-b56e-4a59462f0f00"
-API_SECRETE = "sma_FajgH3VbPu68gwy0PzccvhyGRyy1a8CCHhhvy6ooeg1O_"
+API_KEY = "cea8969e-2acd-4ff4-b88b-e6b62d5cd857"
+API_SECRETE = "sma_nBy04LSDJhwxXNct8R8pREkwzbUTWjU4ZmQj7Y2ApAi_"
 
-TENANT = "demo"
+TENANT = "demo-sdk-test"
```

Comment:
Should this value be puled from a const in a const.py file?

Reply:
Added const file to import these variables